src/js/view/controls/controls.js

```diff
@@ -202,8 +202,13 @@ export default class Controls extends Events {
             lastState = state;
 
             const settingsButton = this.controlbar.elements.settingsButton;
-            if (!visible && isKeyEvent && settingsButton) {
-                settingsButton.element().focus();
+
+            if (!visible && settingsButton) {
+                if (isKeyEvent) {
+                    settingsButton.element().focus();
+                } else {
+                    this.playerContainer.focus();
```

Comment:
If you call `playerContainer.focus()` when the player is floating, it will scroll the page up to where the player was before it started floating. When you want to give the active player container focus you have to either focus `playerContainer` or `wrapperElement`. These are both properties of controls, but to see examples of how we choose the correct one, look in view.js.

Reply:
Great feedback, making changes now!

Reply:
Fixed!